JavierPB7-JavierPalomino.tll

```diff
@@ -0,0 +1,14 @@
+@prefix event: <http://www.ontogrid.net/StickyNote#> .
+@prefix relat: <http://www.w3.org/2001/XMLSchema#> .
```

Comment:
Try not to use a different prefix for one that is usually used by default, in this case xsd